client/signup/steps/site-type/index.jsx

```diff
@@ -35,8 +35,9 @@ class SiteType extends Component {
 			hasInitializedSitesBackUrl,
 		} = this.props;
 
-		const headerText = translate( 'Start with a site type' );
-		const subHeaderText = '';
+		const headerText = translate( 'What are we building today?' );
```

Comment:
We should try keep changes like these in separate PRs incase changes for the radio buttons get caught up in reviews.

Reply:
Moved to #31467